testonly/compact_merkle_tree.go

```diff
@@ -0,0 +1,30 @@
+package testonly
+
+import (
+	"github.com/google/trillian"
+)
+
+func GetInputs() []trillian.Hash {
```

Comment:
Might be better to give these and the functions below more specific names now for when they're used by other packages.